contents/testing/integration-testing.md

```diff
@@ -0,0 +1,143 @@
+<frontmatter>
+  title: Integration Testing
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Integration Testing
+
+**Author(s): [Jacob Li PengCheng](https://github.com/jacoblipech)**
+
+Reviewers:
+
+## What is integration testing?
+
+Integration testing is a part of software testing where individual units of your application are combined and tested as a group. This usually happens after the [<span title="Testing of individual components within a system">unit testing stage</span>](http://softwaretestingfundamentals.com/unit-testing/) and before the [<span title="Evaluation of the software against requirements gathered from users and system specifications">system testing stage</span>](https://www.guru99.com/system-testing.html).
+
+## Why is integration testing important to Software Engineering?
+
+In a software application, each software module is usually designed and unit tested by different programmers. Hence, with different understanding of the software requirements, integration testing is necessary to ensure that software modules work in unity and expose any faults in the interaction between different units. 
+
+<box type="info">Suppose you are in charge of the **data collection component** and it passes the **data** collected from the users to the **data analysis component**. If the **data analysis component** assumes your **data** to be of maximum 100 lines but the data you send has no limit on the number of lines, this may result in data loss, inaccurate results and even system errors. Hence, it is the integration test that is supposed to find this discrepancy before any serious damage is made.</box>
```

Comment:
It is meant to be like an info tag but in a box form for people to read if they need an example to be clearer about how integration can help.

Reply:
Since this chapter is targeted at beginners, I think you can assume that the reader will "need an example to be clearer about how integration can help."

In addition, the next paragraph starts with `Hence`, which concludes based on this info box. Right?